README.md

```diff
@@ -60,9 +60,10 @@ By default, the module is configured to read the configuration from ENV variable
 Currently, the following ENV variables are supported:
 
 | ENV variable | Description | Default value |
-| ------------ | ----------- | ------------- |
+| ------------ | ----------- |-------|
 | VOCDONI_API_ENDPOINT_ENV | The environment of the Vocdoni API. Only two values are accepted: `dev`, `stg`. Read more on [Vocdoni SDK Usage Environment](https://github.com/vocdoni/vocdoni-sdk#environment) | `stg` |
-| VOCDONI_MINUTES_BEFORE_START | How many minutes should the setup be run before the election starts | `10` |
+| VOCDONI_MINUTES_BEFORE_START | How many minutes should the setup be run before the election starts | `10`  |
```

Comment:
There's an extra space here that:
```suggestion
| VOCDONI_MINUTES_BEFORE_START | How many minutes should the setup be run before the election starts | `10` |
```

This kind of changes make difficult to work with git tooling (for instance git blame)